### PR TITLE
Modusers 195

### PR DESCRIPTION
**Pull request by jeremythuff**
Resolves [MODUSERS-195](https://issues.folio.org/browse/MODUSERS-195)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>mod-custom-fields</artifactId>
-      <version>1.3.0</version>
+      <version>1.3.1-SNAPSHOT</version>
```
Comment (bohdan-suprun) [inline pom.xml:33]: We don't usually add snapshot versions

Comment (jeremythuff) [reply pom.xml:33]: I agree, this was at the suggestion of the custom field folks, and defiantly was not working without it. Should this update hold until custom-fields is released?

Comment (bohdan-suprun) [reply pom.xml:33]: I'm fine to leave snapshot if it is updated to released version once available. The module can not be released util we have the lib released, I think.

Comment (marcjohnson-kint) [reply pom.xml:33]: @jeremythuff Please can you raise an issue for updating this dependency to a release version and add it as a blocker for releasing mod-users.

File: src/test/java/org/folio/moduserstest/GroupIT.java [hunk 2]
```diff
@@ -363,8 +356,8 @@ public void test2Group(TestContext context) throws Exception {
         + "\nStatus - " + addExpiredUserResponse.code + " at "
         + System.currentTimeMillis() + " for " + addUserURL + " (addExpiredUser)");
       context.assertEquals(addExpiredUserResponse.code, 201);
-      CompletableFuture<Void> getExpirationCF = new CompletableFuture();
-      ExpirationTool.doExpirationForTenant(RestITSupport.vertx(), RestITSupport.context(), "diku").setHandler(
+      CompletableFuture<Void> getExpirationCF = new CompletableFuture<Void>();
+      ExpirationTool.doExpirationForTenant(RestITSupport.vertx(), RestITSupport.context(), "diku").onComplete(
         res -> getExpirationCF.complete(null));
```
Comment (marcjohnson-kint) [inline src/test/java/org/folio/moduserstest/GroupIT.java:361]:
```suggestion
      ExpirationTool.doExpirationForTenant(RestITSupport.vertx(), RestITSupport.context(), "diku")
        .onComplete(res -> getExpirationCF.complete(null));
```

**Comment by marcjohnson-kint** [comment]
@jeremythuff Do you have time to add a test for expiration of users, which is where most of the low coverage warning is coming from?

**Pull request merged**